src/main/java/org/folio/writer/impl/MarcRecordWriter.java

```diff
@@ -59,6 +67,7 @@ public void writeDataField(RecordDataField recordDataField) {
 
   @Override
   public String getResult() {
+    log.info("getResult:: ");
```

Comment:
This line doesn't contain any business-related information. Why do we need to know that namely this method starts execution? Add business related information or at least rename information string with sth like "Method NNN starting..."